src/components/CreateNewProjectWizard/CreateNewProjectWizard.js

```diff
@@ -96,16 +99,46 @@ class CreateNewProjectWizard extends PureComponent<Props, State> {
     }
   };
 
+  checkProjectLocationUsage = () => {
+    return new Promise((resolve, reject) => {
+      const projectName = getProjectNameSlug(this.state.projectName);
+      if (checkIfProjectExists(this.props.projectHomePath, projectName)) {
+        // show warning that the project folder already exists & stop creation
+        dialog.showMessageBox(
+          {
+            type: 'warning',
+            title: 'Project directory exists',
+            message:
+              'Please pick a different location or name.\nYou could also manually delete the project from disk and create the project with that name.',
```

Comment:
OK, I've changed to your copy. I think that's better. 

We could also display the import button if we detected that the directory exists - or is this confusing? I think we could also leave it as it is now but then the user needs to use the application menu to import the project because if the modal will be dismissed & reopened it will display the same state of the Project creation wizard.

Alternatively we could add a reset button to the create project wizard so it's possible to reset to the first creation step.